hazelcast/src/main/java/com/hazelcast/map/impl/record/AbstractRecord.java

```diff
@@ -268,7 +268,7 @@ protected int stripBaseTime(long value) {
         return diff;
     }
 
-    private static long zeroOutMillis(long value) {
+    public static long zeroOutMillis(long value) {
```

Comment:
could we move this to some util class? an operation depending directly on a record is not nice.